### PR TITLE
Adding descriptions to wizard

### PR DESCRIPTION
**Pull request by Matejkys**
Adding descriptions to fields which should be filled by users during Scaffold set up, as without that it might be sometime confusing what to put there.

### CODE REVIEW

File: scaffolds/PaymoTimeTracking/manifest.json [hunk 1]
```diff
@@ -53,7 +54,8 @@
                             "#username": {
                                 "title": "API Token",
                                 "type": "string",
-                                "format": "password"
+                                "format": "password",
+                                "description": "Fill in your [Paymo API Token](https://github.com/paymoapp/api/blob/master/sections/authentication.md#api-keys)."
```
Comment (odinuv) [inline scaffolds/PaymoTimeTracking/manifest.json]: MD afaik does not work in the configuration schema - you might want to try: 
```
"links": [
        {
          "rel": "Get Paymo API Token",
          "href": "https://github.com/paymoapp/api/blob/master/sections/authentication.md#api-keys"
        }
      ],
```

Comment (Matejkys) [reply scaffolds/PaymoTimeTracking/manifest.json]: @odinuv Thanks! I've been trying to find that around the web without any success :) I'll update it.

**Comment by Matejkys** [comment]
@odinuv Hodil jsem tam ty linky (snad jsem strukturu pochopil správně) + jsem opravil překlepy.

**Comment by odinuv** [comment]
@Matejkys merged & released, i've also updated the docs with the links example so you don't have to search the whole internet next time https://developers.keboola.com/extend/component/ui-options/configuration-schema/#links-example

**Pull request merged**